Victor-Ugwueze/MyDiary
# #159310129 Chore setup postgresql database

### What does this PR do ?
This PR adds changed the project directory structure, 
 adds PostgreSQL database and modify the entry model

### Description of task to be completed ?
Implement the following:

Change project directory structure to 
  
       +--server
            + --controllers
            + --helpers
            + --models
            + --routes
            + --tests

Add PostgreSQL database
Create a database Schema for tables
Add database connection to entry model
Update all import paths
Update package,json

### How should this be tested
You should have PostgreSQL installed before you get started.  Then clone this repo, run `npm install` to install all dependencies and devDependencis.
Open server/helpers/dbHelper.js change the connection string to match that of your local machine.
Then run `npm run start-dev`

### What are the relevant pivotal tracker stories?
#159310129

### Any background context you wish to provide?
None
